src/guide/components/slots.md

```diff
@@ -65,11 +65,11 @@ function FancyButton(slotContent) {
 }
 ```
 
-插槽内容不仅仅局限于文本。它也可以是任意合法的模板内容，例如我们可以传入一些元素，甚至是组件：
+不是只有 text 元素可用作插槽内容。插槽内容还可以是任意合法的模板内容，例如我们可以传入多个元素，甚至是组件：
```

Comment:
```suggestion
除了文本以外，插槽内容还可以是任意合法的模板内容。例如我们可以传入多个元素，甚至是组件：
```

原文：
> Slot content is not just limited to text. It can be any valid template content. For example, we can pass in multiple elements, or even other components:

这里我倾向于使用“文本”，而不是“text 元素”。并且句式感觉可以根据中文习惯做一些简化。

Reply:
这个简化很不错